bom/pom.xml

```diff
@@ -536,6 +545,18 @@
             </plugin>
          </plugins>
       </pluginManagement>
+      <plugins>
+         <plugin>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+            <extensions>true</extensions>
+            <configuration>
+               <nexusUrl>${jboss.releases.nexus.url}</nexusUrl>
+               <serverId>${jboss.releases.repo.id}</serverId>
+               <autoReleaseAfterClose>true</autoReleaseAfterClose>
```

Comment:
@slaskawi I wouldn't really want to invoke `nexus-staging-maven-plugin:deploy` separately, my proposal would be to make all our modules inherit from `infinispan-parent` and only import `wildfly-parent` instead of inheriting from it. I'm not sure how easy that would be, but I'm thinking we can always copy the stuff we need from `wildfly-parent` into our own POMs...